base/shell/explorer/startmnusite.cpp

```diff
@@ -154,6 +207,11 @@ class CStartMenuSite :
                        IDM_DOCUMENTS,
                        MF_BYCOMMAND);
         }
+        else
+        {
+            HMENU hDocMenu = FindSubMenu(hMenu, IDM_DOCUMENTS, FALSE);
+            FillDocumentsMenu(hDocMenu);
+        }
```

Comment:
this thing is a hack. there is a more proper way to do that

Reply:
OK.